web-registry/src/lib/ledger.ts

```diff
@@ -6,8 +6,15 @@ export const nctBasket = process.env.REACT_APP_NCT_BASKET;
 // expLedger is a feature flag indicating the app is connected to a testnet with experimental features (Hambach).
 export const expLedger = process.env.REACT_APP_EXP_LEDGER;
 
+// v4Ledger is a feature flag indicating the app is connected to a testnet with Regen Ledger v4.0.
+// It can be removed once hambach/redwood/mainnet upgrades to v4.0.
+export const v4Ledger = process.env.REACT_APP_V4_LEDGER;
```

Comment:
We will be able to delete these temporary V4 env vars. But not necessary right now.